sdcm/es.py

```diff
@@ -34,7 +34,10 @@ def create_doc(self, index, doc_type, doc_id, body):
         LOGGER.info('DOC_ID: %s', doc_id)
         LOGGER.info('BODY: %s', body)
         self._create_index(index)
-        self.create(index=index, doc_type=doc_type, id=doc_id, body=body)
+        if self.exists(index=index, doc_type=doc_type, id=doc_id):
+            self.update(index=index, doc_type=doc_type, id=doc_id, body={'doc': body})
```

Comment:
Will the old info still exist with update?  can we just remove exist one and recreate a new one?

Reply:
It is up to elastic.
I think it is pretty handy to be able to see prior test stats/errors.